apps/apps.humanatlas.io/src/app/components/server-selector/server-selector.component.scss

```diff
@@ -16,12 +17,12 @@
     }
 
     mat-hint {
-      color: var(--mat-sys-primary);
+      color: vars.$primary;
     }
 
     mat-label {
       @include utils.use-font(label, small);
-      color: var(--mat-sys-inverse-surface) !important;
+      color: vars.$inverse-on-surface !important;
```

Comment:
Please make sure the change from `inverse-surface` to `inverse-on-surface` is correct